### DISCUSSION TITLE
Whisper-live

### DISCUSSION THREAD

**Discussion by alexterminator88** [openai/whisper#1955] [General]
Allo

**Comment by makaveli10** [comment]
[WhisperLive](https://github.com/collabora/WhisperLive) is real time and uses faster-whisper backend.

**Comment by Gldkslfmsd** [comment]
Hi, 
what streaming algorithm do you use? Are you aware of [Whisper-Streaming](https://github.com/ufal/whisper_streaming)? https://github.com/openai/whisper/discussions/1978
